src/index.cpp

```diff
@@ -467,9 +480,13 @@ size_t Index<T, TagT, LabelT>::load_data(std::string filename)
     }
 
 #ifdef EXEC_ENV_OLS
+
+    // REFACTOR TODO: Must figure out how to support aligned reader in a clean manner.
     copy_aligned_data_from_file<T>(reader, _data, file_num_points, file_dim, _aligned_dim);
```

Comment:
This uses the DLVS aligned reader to read the file. It may not be required in the GH code at all. We might have to come up with a FileManager hierarchy to deal with this, so punting it for now.